nav2_collision_monitor/src/collision_monitor_node.cpp

```diff
@@ -68,11 +72,16 @@ CollisionMonitor::on_configure(const rclcpp_lifecycle::State & /*state*/)
     std::bind(&CollisionMonitor::cmdVelInCallback, this, std::placeholders::_1));
   cmd_vel_out_pub_ = this->create_publisher<geometry_msgs::msg::Twist>(
     cmd_vel_out_topic, 1);
+
   if (!state_topic.empty()) {
     state_pub_ = this->create_publisher<nav2_msgs::msg::CollisionMonitorState>(
       state_topic, 1);
   }
 
+  if (visualize_collision_points) {
+    collision_points_marker_pub_ = this->create_publisher<visualization_msgs::msg::MarkerArray>(
+      "~/collision_points_marker", 1);
```

Comment:
```suggestion
      "collision_points_marker", 1);
```

Is the same thing

Reply:
Not really, `~/collision_points_marker` will result in `/<node_name>/collision_points_marker`. `collision_points_marker` will result in `/collision_points_marker`. Since all other topics are without `~/`, I'll remove it

Reply:
Actually I just ran into a problem with this: I have both a collision detector and monitor running and since they have the same topic name, not having the namespace makes them publish to the same topic. I would add the `~/` if you don't object